### ISSUE TITLE
EP062000, fstadv: error x and y are different lengths

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#58]
```r
df <- get_storms(year = 2000, basin = "EP")
al062000 <- df %>% slice(6) %>% .$Link %>% get_fstadv()
```

Error:

```
Error: `.x` (0) and `.y` (2) are different lengths
```

Traceback:

```
32.
purrr::map2(., 1:length(z), function(a, b) {
    newnames <- paste0(names(z[[b]]), z[[b]][[1]])
    stats::setNames(z[[b]], newnames)
}) 
31.
function_list[[k]](value) 
30.
withVisible(function_list[[k]](value)) 
29.
freduce(value, `_function_list`) 
28.
`_fseq`(`_lhs`) 
27.
eval(quote(`_fseq`(`_lhs`)), env, env) 
26.
eval(quote(`_fseq`(`_lhs`)), env, env) 
25.
withVisible(eval(quote(`_fseq`(`_lhs`)), env, env)) 
24.
z %>% purrr::map2(1:length(z), function(a, b) {
    newnames <- paste0(names(z[[b]]), z[[b]][[1]])
    stats::setNames(z[[b]], newnames)
}) at fstadv.R#483
23.
fstadv_forecast_wind_radius(tmp[[b]]$wf) at fstadv.R#393
22.
.f(.x[[i]], .y[[i]], ...) 
21.
purrr::map2(., 1:n, function(a, b) {
    if (tmp[[b]]$wf != "") {
        wf <- fstadv_forecast_wind_radius(tmp[[b]]$wf)
        tmp[[b]] <- dplyr::bind_cols(tmp[[b]], wf) ... 
20.
function_list[[k]](value) 
19.
withVisible(function_list[[k]](value)) 
18.
freduce(value, `_function_list`) 
17.
`_fseq`(`_lhs`) 
16.
eval(quote(`_fseq`(`_lhs`)), env, env) 
15.
eval(quote(`_fseq`(`_lhs`)), env, env) 
14.
withVisible(eval(quote(`_fseq`(`_lhs`)), env, env)) 
13.
tmp %>% purrr::map2(1:n, function(a, b) {
    if (tmp[[b]]$wf != "") {
        wf <- fstadv_forecast_wind_radius(tmp[[b]]$wf)
        tmp[[b]] <- dplyr::bind_cols(tmp[[b]], wf) ... at fstadv.R#390
12.
fstadv_forecasts(contents, date) at fstadv.R#249
11.
.f(.x[[i]], ...) 
10.
purrr::map(filter_fstadv(products), fstadv) at fstadv.R#157
9.
get_fstadv(.) 
8.
function_list[[k]](value) 
7.
withVisible(function_list[[k]](value)) 
6.
freduce(value, `_function_list`) 
5.
`_fseq`(`_lhs`) 
4.
eval(quote(`_fseq`(`_lhs`)), env, env) 
3.
eval(quote(`_fseq`(`_lhs`)), env, env) 
2.
withVisible(eval(quote(`_fseq`(`_lhs`)), env, env)) 
1.
df %>% slice(6) %>% .$Link %>% get_fstadv() 

```

**Comment by timtrice** [comment]
In some advisories such as [EP062000 Adv 45](http://www.nhc.noaa.gov/archive/2000/mar/MEP0600.045.html), wind field radii are appended with "DISSIPATING" or "DISSIPATED"  in place of valid wind field values. 

Added replacement to make these and empty values NA. 

Added tests; passing.

Resolved in commit ce93547.

**Comment by timtrice** [comment]
This issue was resolved at some point in branch develop-0.1.1 when doing audit. Have since been able to write [data file](https://github.com/timtrice/rrricanesdata/tree/master/2000/EP) without warning/error.
